### PR TITLE
Fr us 19

### LINKED ISSUE #22: User Story 19, User Profile Show Page
As a registered user
When I visit my profile page
Then I see all of my profile data on the page except my password
And I see a link to edit my profile data

### PR DESCRIPTION
**Pull request by fredrondina**
completed US 19 plus tests. User profile info now appears on /profile

closes #22

### CODE REVIEW

File: spec/features/users/show_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      expect(page).to have_content(@user1.user_detail.state)
+      expect(page).to have_content(@user1.user_detail.zip_code)
+
+      expect(page).to have_link("Edit Profile")
+
+    end
+    after(:all) do
+      UserDetail.destroy_all
+      User.destroy_all
+    end
```
Comment (brian-greeson) [inline spec/features/users/show_spec.rb:39]: Good to have this here

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<h1>User Profile</h1>
+<p>Name: <%=current_user.user_detail.name%></p>
+<p>Email: <%=current_user.email_address%></p>
+<p>Name: <%=current_user.user_detail.street_address%></p>
+<p>Name: <%=current_user.user_detail.state%></p>
+<p>Name: <%=current_user.user_detail.city%></p>
+<p>Name: <%=current_user.user_detail.zip_code%></p>
+
```
Comment (brian-greeson) [inline app/views/users/show.html.erb]: Please change the labels to match the data

**Review by brian-greeson — CHANGES_REQUESTED** [review]
Looks good but please change the labels on the profile view *see comment*

File: app/views/users/show.html.erb (hunk 2 above)
Comment (fredrondina) [reply app/views/users/show.html.erb]: Great Catch

Comment (fredrondina) [reply app/views/users/show.html.erb]: requested changes have been made

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
 <p>Name: <%=current_user.user_detail.name%></p>
 <p>Email: <%=current_user.email_address%></p>
-<p>Name: <%=current_user.user_detail.street_address%></p>
-<p>Name: <%=current_user.user_detail.state%></p>
-<p>Name: <%=current_user.user_detail.city%></p>
-<p>Name: <%=current_user.user_detail.zip_code%></p>
+<p>Address: <%=current_user.user_detail.street_address%></p>
+<p>City: <%=current_user.user_detail.city%></p>
+<p>State: <%=current_user.user_detail.state%></p>
+<p>Zip: <%=current_user.user_detail.zip_code%></p>
```
Comment (brian-greeson) [inline app/views/users/show.html.erb:7]: Looks much better now!

**Review by brian-greeson — APPROVED** [review]
A+ Good work!

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<h1>User Profile</h1>
+<p>Name: <%=current_user.user_detail.name%></p>
+<p>Email: <%=current_user.email_address%></p>
+<p>Address: <%=current_user.user_detail.street_address%></p>
+<p>City: <%=current_user.user_detail.city%></p>
```
Comment (kelshadarby) [inline app/views/users/show.html.erb:5]: Very easy to read!

**Review by kelshadarby — APPROVED** [review]

**Pull request merged**